kolibri/core/content/utils/importability_annotation.py

```diff
@@ -34,14 +35,15 @@ def get_channel_annotation_stats(channel_id, checksums=None):
     ContentNodeTable = bridge.get_table(ContentNode)
     FileTable = bridge.get_table(File)
     LocalFileTable = bridge.get_table(LocalFile)
-
     if checksums is not None:
         file_table = FileTable.join(
             LocalFileTable,
             and_(
                 FileTable.c.local_file_id == LocalFileTable.c.id,
                 or_(
-                    LocalFileTable.c.id.in_(checksums),
+                    # checksums are not uuids and have been got from
+                    # get_channel_stats_from_disk, so no need to validate them:
+                    filter_by_uuids(LocalFileTable.c.id, checksums, validate=False),
```

Comment:
It's not the validation that's the issue - I think that's still useful in this case, because the validation helps to santize the input, but it's the formatting the checksums as `uuids` for postgresql that is the issue.